Sources/ViewInspector/ViewSearchIndex.swift

```diff
@@ -91,19 +91,19 @@ internal extension ViewSearch {
             .first(where: { $0.viewType.namespacedPrefixes.contains(fullName) }) {
             return identity
         }
-        if (try? content.extractCustomView()) != nil,
-           let inspectable = content.view as? Inspectable {
+        if (try? content.extractCustomView()) != nil {
             let name = Inspector.typeName(
                 value: content.view, generics: .customViewPlaceholder)
-            switch inspectable.entity {
-            case .view:
+            switch content.view {
+            case _ as any View:
                 return .init(ViewType.View<ViewType.Stub>.self, genericTypeName: name)
-            case .viewModifier:
+            case _ as any ViewModifier:
                 return .init(ViewType.ViewModifier<ViewType.Stub>.self, genericTypeName: name)
-            case .gesture:
+            case _ as any Gesture:
+                break
+            default:
```

Comment:
We add a `default` case but we haven't made this code any less "safe". Previously we would return `nil` if `content.view` did not conform to `Inspectable`.